docs/api-swagger-shorter.yaml

```diff
@@ -207,89 +207,12 @@ paths:
           $ref: '#/components/responses/OtherErrors'
       security:
         - bearerAuth: []
-  /auth/callback:
-    post:
-      tags:
-        - SSO
-      summary: Exchange authorization code for tokens
-      description: Exchanges an authorization code for access and refresh tokens.
-      operationId: exchangeAuthCode
-      requestBody:
-        description: Authorization code received from the authentication provider.
-        content:
-          application/json:
-            schema:
-              $ref: '#/components/schemas/AuthCallbackBody'
-        required: true
-      responses:
-        "200":
-          description: Tokens issued successfully
-          content:
-            application/json:
-              schema:
-                $ref: '#/components/schemas/AuthTokens'
-        "400":
-          $ref: '#/components/responses/BadRequest'
-        "401":
-          $ref: '#/components/responses/Unauthorized'
-        "500":
-          $ref: '#/components/responses/OtherErrors'
-      security: []
-  /auth/refresh:
-    post:
-      tags:
-        - SSO
-      summary: Refresh tokens
-      description: Refreshes the access token using a valid refresh token.
-      operationId: refreshToken
-      requestBody:
-        description: Refresh token required for generating a new access token.
-        content:
-          application/json:
-            schema:
-              $ref: '#/components/schemas/AuthRefreshBody'
-        required: true
-      responses:
-        "200":
-          description: Token refreshed successfully
-          content:
-            application/json:
-              schema:
-                $ref: '#/components/schemas/AuthTokens'
-        "400":
-          $ref: '#/components/responses/BadRequest'
-        "401":
-          $ref: '#/components/responses/Unauthorized'
-        "500":
-          $ref: '#/components/responses/OtherErrors'
-      security: []
-  /auth/logout:
-    post:
-      tags:
-        - SSO
-      summary: Logout user
-      description: Logs out the user and invalidates their session tokens.
-      operationId: logoutUser
-      responses:
-        "200":
-          description: User logged out successfully
-          content:
-            application/json:
-              schema:
-                $ref: '#/components/schemas/Status'
-        "401":
-          $ref: '#/components/responses/Unauthorized'
-        "500":
-          $ref: '#/components/responses/OtherErrors'
-      security:
-        - bearerAuth: []
   /auth/status:
     get:
       tags:
         - SSO
       summary: Check login status
-      description: Checks whether the user is currently logged in and retrieves session
-        details and account privileges.
+      description: Checks whether the user is currently logged in and retrieves session details and account privileges.<br>https://samuelus.atlassian.net/wiki/spaces/ZPI/pages/28147713/Dokumentacja+Struktury+JSON+dla+Uprawnie
```

Comment:
I think this link is mostly useless, because only people logged into our jira can access it. If it is just for us, then ok.

Reply:
*General comment

Reply:
It's a line from API v0.0.4 xd. I think it's worth as a documentation for us, because it can't be explained as wide as it should in this short description.
But this documentation needs changes to, so I see if it;s possible to shrink it and post all of it here.